package.json

```diff
@@ -81,6 +81,7 @@
     "@nestjs/sequelize": "8.0.0",
     "@nestjs/swagger": "4.8.2",
     "@nestjs/terminus": "7.0.1",
+    "@nrwl/next": "13.10.5",
```

Comment:
Hm. Did the NX generator add this as `dependencies` rather than a `devDependencies`?

It should at least only be listed once.

Reply:
Yep, the NX generator added this

Reply:
I removed it from the dependencies object